app/components/header_component.html.slim

```diff
@@ -21,6 +19,8 @@
       .dfe-header__menu
         button#toggle-menu.dfe-header__menu-toggle aria-controls='header-navigation' aria-expanded='false' data-action='click->reveal#toggle'
           | Menu
+  = tag.div(class: 'dfe-width-container dfe-header__service-name')
```

Comment:
Why use Ruby for the wrapper?

Reply:
I have updated this so it is just the div and class on this line